go.mod

```diff
@@ -9,7 +9,4 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
-require (
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
-	golang.org/x/text v0.3.7 // indirect
```

Comment:
🤔 it's not clear to me why `golang.org/x/text` was automatically removed. It seems unrelated to this change.

Reply:
Reverted.